### DISCUSSION TITLE
123

### DISCUSSION THREAD

**Discussion by jojoldu** [jojoldu/blog-comments#742] [Comments]
http://jojoldu.tistory.com/123

**Comment by jojoldu** [comment]
작성자: ~

잘보았습니다.

**Comment by jojoldu** [comment]
작성자: lamune

직접 따라 하면서 많은 도움이 되었습니다. 
감사합니다. ^-^ 

궁금한 점이 있는데요. 
module-web 의 build.gradle의 설정은 어떻게 해 주어야 할까요?

**Comment by joont92** [comment]
잘봤습니다 감사합니다.
중간에 테스트 코드 작성하는 부분에서 궁금한게 있는데요..

스프링부트는 실행전에 클래스패스에서 `@SpringBootApplication`을 찾고 있으면 로드하고,
로드하면서 빈 스캐닝을 한다.. 그래서 `@SpringBootApplication`이 필요하다. 라고 보면 될까요?

**Comment by ghost** [comment]
안녕하세요. 모듈 이름에 대해서 궁금한 점이 있는데요.
인텔리J에서 설명하는 글이나 여기서 설명해주신 것에 보면 모듈의 이름에 '-'이 들어가 있는데, 패키지랑은 별도로 봐야 하는 건가요? 하이픈이 들어가는 게 웹 쪽에서 많이 쓰는 건지 Github나 인텔리j 쪽에서 쓰는 방식인지 궁금합니다.

**Comment by ahndy84** [comment]
이글을 읽고 암이 나았습니다. :+1:

**Comment by catniz** [comment]
이 글을 읽고 암이 나았습니다. 2222

**Comment by youngwonseo** [comment]
이 글을 읽고 암이 나았습니다. 3333
